packages/engine/README.md

```diff
@@ -182,34 +224,35 @@ If one of the environment variables shown in the help page is passed, it will ov
 
 > **Warning** - Python and Rust runners are currently not supported. Within your simulation project, you should only see `.js` files within dependencies (for example, dependencies/@hash/age/src/behaviors/age.js). Files ending in `.rs` and `.py` will be ignored and the run will possibly fail in unclear ways. This also means that only an `init.json` or `init.js` is supported at the moment, not `init.py`.
 >
-> Currently, the easiest way of creating a project is by using the integrated IDE at [https://core.hash.ai][hCore] (hCore). In the absence of an in-depth description of expected project structure (which will be coming in the future), downloading a project from hCore is currently the easiest way to learn how one should be set out.
+> Currently, the easiest way of creating a project is by using the integrated IDE at [https://core.hash.ai][hcore] (hCore). In the absence of an in-depth description of expected project structure (which will be coming in the future), downloading a project from hCore is currently the easiest way to learn how one should be set out.
 
 In order to download and then run a simulation from hCore, you can download it by clicking `File -> Export Project` from the toolbar on the top. For help in finding or creating, and editing, simulations in hCore, take a look at our [online documentation][docs]. Then save and unzip the downloaded project to a location of your choice, for example by
 
 ```shell
-$ unzip my-project.zip -d my-hash-project
+unzip my-project.zip -d my-hash-project
```

Comment:
`$` removed by `prettier-plugin-sh`

Reply:
As we don't ever need `#` in our scripts I like this change.